admin/class-nginx-helper-admin.php

```diff
@@ -281,6 +281,28 @@ public function nginx_helper_default_settings() {
 		);
 
 	}
+    
+    public function store_default_options() {
+        $options = get_site_option( 'rt_wp_nginx_helper_options', array() );
+        $default_settings = $this->nginx_helper_default_settings();
+        
+        $removable_default_settings = array(
```

Comment:
Why are we removing these settings from the list of settings to be stored in the options table?